msteams-platform/messaging-extensions/api-based-overview.md

```diff
@@ -411,8 +409,14 @@ Update app manifest (previously called Teams app manifest) with the `composeExte
   "accentColor": "#FFFFFF",
   "composeExtensions": [
     {
-      "composeExtensionType": "apiBased",
-      "apiSpecificationFile": "aitools-openapi.yml",
++      "composeExtensionType": "apiBased",
++      "authorization": {
++        "authType": "apiSecretServiceAuth ",
```

Comment:
extra space in authType, should be "apiSecretServiceAuth"

Reply:
Updated.